babel.config.js

```diff
@@ -0,0 +1,40 @@
+module.exports = (api) => {
+    api.cache(true);
+
+    const presets = [
+        ['@babel/preset-env', {
+            targets: {
+                browsers: ['>5%', 'last 2 versions'],
+            },
+            modules: false,
+        }],
+    ];
+
+    const plugins = [
+        // Allow proper tree shaking of lodash ES6 named imports
+        'lodash',
+        // Previously stage-3 (stages are no longer used in babel 7)
+        // See https://babeljs.io/blog/2018/07/27/removing-babels-stage-presets
+        '@babel/plugin-proposal-json-strings',
```

Comment:
Can you let me know what prompted the new plugins?  I would have assumed we could just take the plugins stage-3 encompassed and add them individually, but there's some missing and some new ones here?  Here's the contents of stage-3 as of the current version we use (6.24.1): https://github.com/babel/babel/blob/f8da5e345798c8a5db98302bfbc2621187b0d4ac/packages/babel-preset-stage-3/src/index.js

Do you recall what specifically prompted the need for `@babel/plugin-proposal-json-strings` and `@babel/plugin-syntax-import-meta`?

And also are we sure we no longer need these other plugins that stage-3 was providing? ```
"babel-plugin-syntax-trailing-function-commas";
"babel-plugin-transform-async-to-generator";
"babel-plugin-transform-exponentiation-operator";
"babel-plugin-transform-async-generator-functions";
```

Reply:
There was a console warning that listed out the plugins that listed the plugins to use that linked here, https://github.com/babel/babel/blob/master/packages/babel-preset-stage-3/README.md.

It's possible we don't need `@babel/plugin-syntax-import-meta` and some of the other ones, but while I was testing it I opted to drop in the suggested "same configuration".

Reply:
Ah, ok.  That makes sense.  I bet those are what was stage-3 as of `v7.0.0-beta.55`, which would be different than what was stage-3 in our `6.24.1` version.  Let's try adding the  ones from `6.24.1` above and removing any others from `7.0.0-beta.55`.

Then it's probably worth a quick investigation if we even need all of the 6.24.1 stage-3 plugins, it's highly likely we added that for rest/spread and just got the others but didn't actually need/use them

Reply:
>Without any configuration options, @babel/preset-env behaves exactly the same as @babel/preset-es2015, @babel/preset-es2016 and @babel/preset-es2017 together (or the deprecated babel-preset-latest).

This means we don't need to add them back, but I will remove the ones we may not need.